docs/pages/guides/monorepos.md

```diff
@@ -190,6 +280,39 @@ export default function App() {
 
 As mentioned earlier, using monorepos is not for everyone. You take on increased complexity and need to solve issues you most likely will run into. Here are a couple of common issues you might encounter.
 
+### Can I use another monorepo tool instead of Yarn workspaces?
+
+There are a lot of monorepo tools available, and each of these tools has its benefits. It's hard for us to keep up with the latest tools and methods, and because of that, we can't officially support new monorepo tools. That being said, if the tool follows these three rules, it should work fine.
+
+<Collapsible summary="1. All dependencies must be installed in a node_modules directory">
+
+React Native dependencies contain many other files besides JavaScript, like Gradle files such as `react-native/react.gradle`. These native files are referenced from different sources other than Node, and because of that, it makes it fundamentally incompatible with concepts like Plug'n'Play modules.
+
+</Collapsible>
+
+<Collapsible summary="2. Dependencies used in multiple workspaces can be installed in the root node_modules directory">
+
+Whenever multiple workspaces use the same version of a single dependency, they can be installed in a root **node_modules** directory. Monorepo tools usually do this to remove duplicate tasks, like installing the exact same dependency twice in different places. This rule isn't necessary but does set us up for rule #3.
```

Comment:
```suggestion
Whenever multiple workspaces use the same version of a single dependency, they can be installed in a root **node_modules** directory. Monorepo tools usually do this to remove duplicate tasks, like installing the same dependency twice in different places. This rule isn't necessary but does set us up for rule #3.
```

Reply:
I think `exact` here emphasizes the exact similarity of the packages. Here, a package should only be considered "same" if the name _and_ exact version are equal. If a different version is used, it shouldn't be the same dependency.

Do you think we should still remove the `exact` or use something else to highlight this better? @amandeepmittal